modules/microshift-cri-o-container-runtime.adoc

```diff
@@ -26,9 +39,32 @@ Environment=HTTPS_PROXY="http://$PROXY_USER:$PROXY_PASSWORD@$PROXY_SERVER:$PROXY
 $ sudo systemctl daemon-reload
 ----
 
-. Restart the CRI-O service to apply the settings:
+. Restart the CRI-O service:
 +
 [source,terminal]
 ----
 $ sudo systemctl restart crio
 ----
+
+. Restart the {microshift-short} service to apply the settings:
++
+[source,terminal]
+----
+$ sudo systemctl restart microshift
+----
+
+.Verification
+
+. Verify that pods are started by running following command and examining the output:
++
+[source,terminal]
+----
+$ oc get all -A
+----
+
+. Verify that {microshift-short} is able to pull container images by running the following command and examining the output:
++
+[source,terminal]
+----
+$ sudo crictl images
+----
```

Comment:
Do we have example output for a successful configuration?

Reply:
No, I didn't want to use the customer's output from the support case. I can ask QE if there's something I can list here.